crates/bevy_render/src/camera/projection.rs

```diff
@@ -51,6 +51,17 @@ pub enum WindowOrigin {
     BottomLeft,
 }
 
+#[derive(Debug, Clone, Reflect, Serialize, Deserialize)]
+#[reflect_value(Serialize, Deserialize)]
+pub enum ScalingMode {
+    // Match the window size. 1 world unit = 1 pixel.
```

Comment:
Is there any reason these aren't doc comments?

Reply:
no, that was an accident. i'll fix it